src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.Bytes.cs

```diff
@@ -53,23 +47,26 @@ private void WriteBase64StringHelper(ReadOnlySpan<byte> utf8PropertyName, ReadOn
         /// <exception cref="InvalidOperationException">
         /// Thrown if this would result in an invalid JSON to be written (while validation is enabled).
```

Comment:
Some of the feedback was missed:
https://github.com/dotnet/corefx/pull/38864#discussion_r297200606

> "would result in an invalid JSON to be written" is strange grammatically. How about "would result in invalid JSON being written"?

Reply:
Fixed. Good catch.

Reply:
The exception text here should be the condition when the exception is thown without if's, etc.

```suggestion
        /// An invalid JSON would be written (when validation is enabled).
```

Reply:
Or even better, avoid would and use why this happened like "Invalid JSON format provided and validation is enabled."

Reply:
Would the same feedback apply to other exception messages (outside this PR) like:
https://github.com/dotnet/corefx/blob/05c0f516cf20cd6ae108ba034984b0cdc740732e/src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.Bytes.cs#L47-L49
```text
Thrown when the specified property name is too large.
```

If so, how would you word that?
We have this type of phrasing in a lot of exception comments within the JSON stack so probably its outside the scope of this PR, but would be good to address.

Reply:
Yes, we've been fixing those when we get the docs from @carlossanlop on the PRs. But it would be good to get them correct from source.
In this case, the right change would be

`<paramref name="propertyName"/> is <see langword="null"/>`

Our wiki has some guidance on that: https://github.com/dotnet/dotnet-api-docs/wiki/Exceptions